### ISSUE TITLE
Doesn't Install

### ISSUE THREAD

**Issue by alhassanv** [jsmecham/atom-terminal-tab#122]
When clicking "install" after a small wait, it goes back to the same button.

**Comment by alhassanv** [comment]
Here is the log

**Comment by jsmecham** [comment]
I just published 0.6.0, please try again and let me know if it is still failing.

**Comment by HebaruSan** [comment]
Failed install log for 0.6.0 on Atom 1.41.0 on Ubuntu 19.10:

```
> node-pty-prebuilt-multiarch@0.9.0 install /tmp/apm-install-dir-1191110-28291-1tiwfm0.tb3s/node_modules/terminal-tab/node_modules/node-pty-prebuilt-multiarch
> prebuild-install || node scripts/install.js


Aborted (core dumped)
Aborted (core dumped)
gyp: Call to 'node -e "require('nan')"' returned exit status 134 while in binding.gyp. while trying to load binding.gyp
gyp ERR! configure error 
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/lib/configure.js:345:16)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:237:12)
gyp ERR! System Linux 5.3.0-23-generic
gyp ERR! command "/usr/share/atom/resources/app/apm/bin/node" "/usr/share/atom/resources/app/apm/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /tmp/apm-install-dir-1191110-28291-1tiwfm0.tb3s/node_modules/terminal-tab/node_modules/node-pty-prebuilt-multiarch
gyp ERR! node -v v10.2.1
gyp ERR! node-gyp -v v3.7.0
gyp ERR! not ok 
Aborted (core dumped)
Aborted (core dumped)
npm WARN enoent ENOENT: no such file or directory, open '/tmp/apm-install-dir-1191110-28291-1tiwfm0.tb3s/package.json'
npm WARN apm-install-dir-1191110-28291-1tiwfm0.tb3s No description
npm WARN apm-install-dir-1191110-28291-1tiwfm0.tb3s No repository field.
npm WARN apm-install-dir-1191110-28291-1tiwfm0.tb3s No README data
npm WARN apm-install-dir-1191110-28291-1tiwfm0.tb3s No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 134
npm ERR! node-pty-prebuilt-multiarch@0.9.0 install: `prebuild-install || node scripts/install.js`
npm ERR! Exit status 134
npm ERR! 
npm ERR! Failed at the node-pty-prebuilt-multiarch@0.9.0 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.atom/.apm/_logs/2019-12-10T17_32_03_007Z-debug.log
```

[2019-12-10T17_32_03_007Z-debug.zip](https://github.com/jsmecham/atom-terminal-tab/files/3946359/2019-12-10T17_32_03_007Z-debug.zip)
